onchain/rollups/contracts/consensus/quorum/Quorum.sol

```diff
@@ -3,32 +3,44 @@
 
 pragma solidity ^0.8.8;
 
-import {AccessControlEnumerable} from "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
-import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 import {PaymentSplitter} from "@openzeppelin/contracts/finance/PaymentSplitter.sol";
 
 import {AbstractConsensus} from "../AbstractConsensus.sol";
+import {IConsensus} from "../IConsensus.sol";
 import {IHistory} from "../../history/IHistory.sol";
 
 /// @title Quorum consensus
 /// @notice A consensus model controlled by a small set of addresses, the validators.
-/// @dev This contract uses several OpenZeppelin contracts:
-/// `AccessControlEnumerable`, `EnumerableSet`, and `PaymentSplitter`.
-/// For more information on those, please consult OpenZeppelin's official documentation.
-contract Quorum is AbstractConsensus, AccessControlEnumerable, PaymentSplitter {
-    using EnumerableSet for EnumerableSet.AddressSet;
-
-    /// @notice The validator role.
-    /// @dev Only validators can submit claims.
-    bytes32 public constant VALIDATOR_ROLE = keccak256("VALIDATOR_ROLE");
-
+///         In this version, the validator set is immutable.
+/// @dev This contract uses OpenZeppelin `PaymentSplitter`.
+///      For more information on `PaymentSplitter`, please consult OpenZeppelin's official documentation.
+contract Quorum is AbstractConsensus, PaymentSplitter {
     /// @notice The history contract.
     /// @dev See the `getHistory` function.
     IHistory internal immutable history;
 
-    /// @notice For each claim, the set of validators that agree
-    /// that it should be submitted to the history contract.
-    mapping(bytes => EnumerableSet.AddressSet) internal yeas;
+    // Quorum members
+    // Map an address to true if it's a validator
+    mapping(address => bool) public validators;
```

Comment:
Okay, fair enough. But then we don't need to annotate this state variable with "Quorum members".

Reply:
So, although `validators` denotes a set, indexing this variable and getting a boolean looks a lot like an $\in$-test, so I think it would the code that uses this variable would look more readable if this was renamed as `isValidator`.

Reply:
yes I do agree with `isValidator` 👍

Reply:
But the thing is we are modifying this var to `mapping(address => uint256) validatorIndex;` in the further optimized version. So `isValidator` is gonna be changed to `validatorIndex` anyways, right?

Reply:
True. :-)